common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -291,16 +292,13 @@ public boolean loadConfFromFile(String fileName, List<ConfigOption<Object>> conf
     if (properties == null) {
       return false;
     }
+    Map<String, String> propertiesFromSystem = new HashMap<>();
     System.getProperties().stringPropertyNames().stream()
         .forEach(
             propName -> {
-              properties.put(propName, System.getProperty(propName));
+              propertiesFromSystem.put(propName, System.getProperty(propName));
             });
-    return loadCommonConf(properties) && loadConf(properties, configOptions, true);
-  }
-
-  public boolean loadCommonConf(Map<String, String> properties) {
```

Comment:
Will deleting this part make it impossible to set the configuration items defined in RssBaseConf?

Reply:
No, I just tested it yesterday, the RssBaseConf is super class of all its sub-class, all its field can be collected by its sub-class.

So this `loadCommonConf` is redundant for now.